packages/core/src/_typography.scss

```diff
@@ -5,7 +5,7 @@
 
 @import "common/variables";
 @import "common/mixins";
-@import "common/icons";
+@import "../../icons/src/icons";
```

Comment:
Need to import from the package; can't use a relative path.

Reply:
...I would have expected this too, but nobody does this anywhere. Everyone just imports from core directly rather than from the package.